ui/source/js/components/controls.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+function Controls(props) {
+
+    function setViewerState(viewerId, controlId) {
```

Comment:
This is a somewhat odd name for this function, since it's not actually setting anything.

Reply:
Agreed